tools/elevate_user.py

```diff
@@ -0,0 +1,115 @@
+import argparse
+from baselayer.app.env import load_env
+from baselayer.app.models import init_db
+from skyportal.model_util import add_user, setup_permissions, role_acls
+from baselayer.app.models import User
+from baselayer.app.models import DBSession
+import sqlalchemy as sa
+
+env, cfg = load_env()
+init_db(**cfg['database'])
+
+parser = argparse.ArgumentParser(
+    description='Elevate user to super admin', add_help=False
+)
+parser.add_argument('--username', help='Username of the user to elevate')
```

Comment:
```suggestion
parser.add_argument('--username', help='User to set role for')
```

Reply:
This is not very important, but thinking about it, perhaps we want:

```
./set_user_role.py user role  -> set user role
./set_user_role.py -> list all users
```

But, not a big deal.

Reply:
@stefanv maybe we keep the `--list` argument, but if none is given we default to also list the users, rather than show the help ?

Reply:
I already added it. If you meant something else, let me know and i'll revert that change.